.docker/Dockerfile.govcms7

```diff
@@ -1,9 +1,34 @@
-FROM govcmsdev/govcms7
```

Comment:
Is there a benefit to this?  At least with govcmsdev/govcms7 there isn't an opportunity for divergence from the named release?